owncloudApp/src/main/java/com/owncloud/android/files/FileMenuFilter.java

```diff
@@ -337,6 +338,24 @@ private boolean isSingleFile() {
         return isSingleSelection() && !mFiles.get(0).isFolder();
     }
 
+    private boolean isAnyFolder() {
+        for (int i = 0; i < mFiles.size(); i++) {
+            if (mFiles.get(i).isFolder()) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    private boolean areDownloaded() {
+        for (int i = 0; i < mFiles.size(); i++) {
+            if (!mFiles.get(i).isDown()) {
+                return false;
+            }
+        }
+        return true;
+    }
+
```

Comment:
`containtsFolder()` works for me. `anyFileDown()` I don´t think it could work because I need to know if any of the files selected is not downloaded.  `anyFileDown()` is the opposite. If I have one file down and the other one not, return true, and if both are down returns true too.